### PR TITLE
Note Addition

### PR DESCRIPTION
**Pull request by joinimran**
As suggested, added a note in the documents regarding usage of 1903 settings in 1909 version as 1909 is incremental version of 1903.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5930

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

CC @e0i

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ This article describes the network connections that Windows 10 components make t
...
 
 >[!IMPORTANT]
+> - The downloadable 1903 scripts/settings can be used on 1909 devices.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:33]:
```suggestion
> - The downloadable Windows 10, version 1903 scripts/settings can be used on Windows 10, version 1909 devices.
```

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by e0i** [comment]
@denisebmsft The PR has been approved by its author and can be merged. Thank you.

**Pull request merged**